ui/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "ui",
```

Comment:
@laurafitzgerald just noticed this now - should we rename this from `ui` to `mobile-security-service-ui`? (maybe we don't need to, I'm just not sure)